projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -536,6 +536,9 @@ public Container getContainer(String containerName) {
 
   /** Return a {@link Container container} contains all testrigs directories inside it */
   public Container getContainer(Path containerDir) {
+    if (!Files.exists(containerDir.resolve(BfConsts.RELPATH_TESTRIGS_DIR))) {
+      return Container.of(containerDir.toFile().getName(), new TreeSet<>());
+    }
```

Comment:
Probably initializing a container should initialize any required subfolders (like testrigs).